NIPostsReload/Common/Extensions/UIViewController/UIViewController+CompositionalLayout.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  UIViewController+CompositionalLayout.swift
+//  NIPostsReload
+//
+//  Created by Denys Niestierov on 29.12.2023.
+//
+
+import UIKit
+
+extension UIViewController {
+    func createCollectionViewItem(
```

Comment:
This is not the right place for a layout method. Not every controller has a collection, so it should not be available for all controllers